package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Mark <mark@remarkablemark.org>",
   "main": "index.js",
   "module": "index.mjs",
+  "exports": "./index.mjs",
```

Comment:
Would this cause a regression for those who use CommonJS?

I see in other [areas](https://stackoverflow.com/questions/58990498/new-package-json-exports-field-not-working-with-typescript) that `import`, `require`, and `types` are defined in `exports`:

```json
{
  "exports": {
    ".": {
      "import": "./index.mjs",
      "require": "./index.js",
      "types": "./index.d.ts"
    }
  }
}
```

Reply:
Nice, hadn't come across that syntax. Updated the PR 😃

Reply:
Thanks!